application/views/template_view.py

```diff
@@ -0,0 +1,17 @@
+from flask import Blueprint, make_response, request, jsonify
+from application.controllers.template_controller import TemplateController
+
+template_view = Blueprint('template_view', __name__)
+
+
+@template_view.route('/upload/<template_id>', methods=['POST'])
```

Comment:
Do we need the /upload/ path if it's a post? It would be more CRUD like if we had the path `/template/id` with a `post` and a `get`. The `post` would be the current upload method and the `get` would exist as it does